<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TiliTapahtumaWindow</class>
 <widget class="QDialog" name="TiliTapahtumaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tilitapahtumat</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tapahtumaTable">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>90</number>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="uudetButton">
     <property name="text">
      <string>Uudemmat tapahtumat</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="aiemmatButton">
     <property name="text">
      <string>Aiemmat tapahtumat</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="takaisinButton">
     <property name="text">
      <string>Takaisin</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
